Orange/tests/test_ada_boost.py

```diff
@@ -11,10 +11,10 @@
 
 
 class TestSklAdaBoostLearner(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.iris = Table("iris")
-        cls.housing = Table("housing")
+    def setUp(self):
```

Comment:
I changed all of these to debug nocopy interaction issues. I'll try to put them back for performance, thanks.